gke-topology-scheduler/schedule-daemon.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: topology-scheduler
+  labels:
+    app: topology-scheduler
+  namespace: kube-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: topology-scheduler
+  template:
+    metadata:
+      labels:
+        app: topology-scheduler
+    spec:
+      tolerations:
+      - key: "node-role.kubernetes.io/control-plane"
+        operator: "Exists"
+        effect: "NoSchedule"
+      containers:
+      - name: topology-scheduler-container
+        image: python:3.10
+        command:
+        - /bin/sh
+        - -c
+        - |
+          pip install google-auth cachetools google-api-python-client kubernetes
+          python -u /scripts/schedule-daemon.py --ignored-namespace kube-system gmp-public gmp-system
```

Comment:
as a follow up, could we build and publish an image instead of downloading dependencies at runtime?

Reply:
I could follow up on building a container shared out. @dburl unless you want to do this as part of the commit.

Reply:
I will add Dockerfile in a separate PR